### DISCUSSION TITLE
Welcome to NLP101 Discussions!

### DISCUSSION THREAD

**Discussion by deeparabicnlp** [deeparabicnlp/NLP101#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deeparabicnlp** [comment]
What is the most active repo for Arabic NLP?

**Reply by jawadatgithub** [reply]
Check this out:
https://github.com/ARBML

**Comment by jawadatgithub** [comment, marked as answer]
Check this out:
https://github.com/ARBML
